src/tools/r2rdump/R2RReader.cs

```diff
@@ -12,6 +12,26 @@
 
 namespace R2RDump
 {
+    public enum Registers
```

Comment:
This enum is X64 specific, but it isn't named as such. It really ought to be in an architecture specific place.